exp-models/addon/models/experiment.js

```diff
@@ -25,11 +25,18 @@ export default DS.Model.extend(JamModel, {
     structure: DS.attr(),
     displayFullscreen: DS.attr('boolean'),
 
+    duration: DS.attr('string'),
+    whatHappens: DS.attr('string'),
+    purpose: DS.attr('string'),
```

Comment:
Are `whatHappens` and `purpose` separate from the existing `description` field? We may want to reorganize the schema and avoid making the fields too specific.